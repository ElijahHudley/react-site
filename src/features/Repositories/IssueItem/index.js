import IssueItem from './issueItem';
export default IssueItem;
