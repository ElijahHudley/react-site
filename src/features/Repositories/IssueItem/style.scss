.issueItem {
  width: 100%;
  height: 100px;
  border-bottom: solid 5px rgba(112, 76, 182, 0.5);
  overflow-y:  auto;

  &:hover {
    background-color: rgba(112, 76, 182, 0.5);
  }
  .on {
    font-size: 16px;
    padding: 16px;
    width: 10%;
    text-align: left;
    margin-right: 8px;
  }

  .off {
    display: none;
  }

  .issueItem-name {
    font-family: 'Courier New', Courier, monospace;
    font-size: 24px;
    text-align: center;
    vertical-align: middle;
    line-height: 90px;
    padding-left: 16px;
  }
}
