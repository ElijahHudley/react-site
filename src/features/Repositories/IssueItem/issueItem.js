import React, {useEffect, useState} from 'react';
import './style.scss';

export default function IssueItem(props) {
    return (<div className={'issueItem'}>
        <span className={'issueItem-name'}>
                <span>{`${props.number}: ${props.name}`}</span>
        </span>
    </div>)
}
