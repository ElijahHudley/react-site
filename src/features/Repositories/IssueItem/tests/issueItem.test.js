import React from 'react';
import renderer from 'react-test-renderer';

import IssueItem from '../issueItem';

describe('test issueItem component', () => {
    test('should match snapshot', () => {
        const tree = renderer
            .create(<IssueItem number={1} name={'test'}>
            </IssueItem>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
