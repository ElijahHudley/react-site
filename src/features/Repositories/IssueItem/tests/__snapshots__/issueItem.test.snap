// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div class={'issueItem'}>
        <span className={'issueItem-name'}>
                <span>1: test</span>
        </span>
    </div>
`;
